### PR TITLE
Changing severity level of prometheus alerts

### PR DESCRIPTION
**Pull request by Red-GV**
This PR updates the Prometheus alerts to have the appropriate severity level.

/cc @periklis 
/assign @ewolinetz 

### Links
- JIRA: https://issues.redhat.com/browse/LOG-1097

### CODE REVIEW

File: hack/prometheus-rules.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ spec:
         ) > 85
       for: 5m
       labels:
-        severity: alert
+        severity: critical
```
Comment (periklis) [inline hack/prometheus-rules.yaml:57]: I suggest to flip the severity `ElasticsearchNodeDiskWatermarkReached == 85%` to `warning` and `ElasticsearchNodeDiskWatermarkReached == 90%` to `critical` here.

**Review by periklis — CHANGES_REQUESTED** [review]
Besides one small suggestion, looks good to me.

File: hack/prometheus-rules.yaml (hunk 1 above)
Comment (Red-GV) [reply hack/prometheus-rules.yaml]: Thanks! I've made the suggested changes.

**Review by periklis — COMMENTED** [review]
/lgtm

**Comment by lukas-vlcek** [comment]
IMO `hack/prometheus-rules.yaml` is garbage and **should be** removed. Prometheus rules are found in `files/prometheus_alerts.yml`.

**Pull request merged**